opflexagent/utils/bridge_managers/ovs_lib.py

```diff
@@ -13,7 +13,9 @@
 import contextlib
 
 from neutron.agent.common import ovs_lib
-from neutron.agent.ovsdb import impl_vsctl
+
+# REVISIT: This is gone in stein!
```

Comment:
Is there a reason we still keep this around?

Reply:
Will fix.